gramps/plugins/lib/libmetadata.py

```diff
@@ -4,6 +4,8 @@
 #
 # Copyright (C) 2011      Nick Hall
 # Copyright (C) 2011      Rob G. Healey <robhealey1@gmail.com>
+# Copyright (C) 2014    Nick Hall
```

Comment:
I'm not sure why another copyright line is added for me.

Reply:
Thanks for the feedback, I will work through it.   The credit to you is for the code that detects the faces that you posted in Mantis (https://gramps-project.org/bugs/view.php?id=7442).  In looking at it now, I see is in the context of the PhotoTagging Gramplet not libmetadata.py

I can merge the lines (Copyright (C) 2011,2014      Nick Hall) or leave them separate or omit 2014.   I leave it to you as to how you want t your contribution recognized.

Reply:
Are you sure it was me?  It looks like Artem Glebov wrote that code.

Reply:
I'm not positive.  I've assumed that Nick_H in Mantis is also Nick-Hall in Github.  I inferred that you added the face detection code because these two posts in Mantis:

- https://gramps-project.org/bugs/view.php?id=7442#c34616
- https://gramps-project.org/bugs/view.php?id=7442#c34651.   'I only attached the file I changed'

Reply:
Yes.  Nick_H is me.  I'd totally forgotten that I'd written that patch.  Thanks for reminding me.

You can just add ", 2014" to my existing line to give me credit.  If you only used my code then there is no need to credit Artem.

Reply:
Done